teuthology/misc.py

```diff
@@ -1373,6 +1374,21 @@ def is_in_dict(searchkey, searchval, d):
         return searchval == val
 
 
+def reimage_fog(args):
+    """
+    Reimage FOG nodes with options specified
+    """
+    machines = args['--nodes']
+    nodes = machines.rstrip(',').split(',')
+    ctx = argparse.Namespace()
+    ctx.os_type = args['--os-type']
+    ctx.os_version = args['--os-version']
+    from teuthology.provision import reimage
```

Comment:
@vasukulkarni can this be resolved in other way than putting import inside methods?

Reply:
We should avoid circular imports.
I would suggest reimage_fog to put to a separate module like others corresponding to scripts or just put it directly to to script, since you're importing teuthology.misc there?